cmd/kube-controllers/main.go

```diff
@@ -54,9 +55,15 @@ func main() {
 	// Install a hook that adds file/line no information.
 	log.AddHook(&logutils.ContextHook{})
 
+	// Tell glog (used by client-go) to log into STDERR. Otherwise, we risk
+	// certain kinds of API errors getting logged into a directory not
+	// available in a `FROM scratch` Docker container, causing glog to abort
+	// hard with an exit code > 0.
+	flag.Set("logtostderr", "true")
```

Comment:
Good point to remember when dealing with K8s and `scratch` for sure. 

Will have to put this one in the old memory banks. 🤔